recipes/lock/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "lock" %}
+{% set version = "2018.3.25.2110" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lock-{{ version }}.tar.gz
+  sha256: cc5ac770930493eed7a8cfd0cf2568a125faf112eb8aa6b6149b3e581523d0c7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+  entry_points:
+    - lock = lock:lock
```

Comment:
What's the deal with this entry point? It's listed in the setup.py as a console script, but it didn't work for me when I installed this package using pip. Is that why you aren't testing it here?

Reply:
The entry point is set here https://github.com/wdbm/lock/blob/master/setup.py#L22.

Without setting it in the feedstock, the build failed on Windows. But now I just realize that this lib can't be used on Windows anyway. So, I can just remove it.

What do you prefer? Keep it here explicitly or remove it?

I don't know much about this package TBH. I just added it because it's a dep of the other packages in that PR.

Reply:
I am trying to figure out whether you forgot to test this entry point or if it was intentionally untested because the entrypoint doesn't look like it is implemented correctly (upstream). I'm guessing you don't intend to use this entry point, so we can leave it as-is and users can open a issue about it later if it doesn't work.